core/drivers/gic.c

```diff
@@ -267,6 +277,32 @@ static void gic_it_disable(struct gic_data *gd, size_t it)
 	write32(mask, gd->gicd_base + GICD_ICENABLER(idx));
 }
 
+static void gic_it_set_pending(struct gic_data *gd, size_t it)
+{
+	size_t idx = it / NUM_INTS_PER_REG;
+	uint32_t mask = 1 << (it % NUM_INTS_PER_REG);
```

Comment:
Shouldn't here keep the same as gic_it_disable? As you recommend, I should change to "BIT32(it % NUM_INTS_PER_REG)"  instead?